src/main/java/com/minecolonies/colony/buildings/BuildingHome.java

```diff
@@ -61,21 +65,18 @@ public void onDestroyed()
     @Override
     public void onWorldTick(TickEvent.WorldTickEvent event)
     {
-        if (event.phase != TickEvent.Phase.END)
-        {
-            return;
-        }
+        if (event.phase != TickEvent.Phase.END) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)